data/formats-data.js

```diff
@@ -7258,11 +7260,13 @@ let BattleFormatsData = {
 	},
 	naganadel: {
 		randomBattleMoves: ["nastyplot", "dragonpulse", "sludgewave", "fireblast", "dracometeor", "uturn"],
+		randomDoubleBattleMoves: ["nastyplot", "dragonpulse", "sludgewave", "sludgebomb", "fireblast", "dracometeor", "uturn", "protect"],
```

Comment:
Is it necessary to give Naganadel Sludge Wave? It isn't even mentioned in the Doubles analysis.

Reply:
nah probably not but I still have the trauma of being murdered by Nasty Plot Naganadel since the double KO caused Beast Boost to trigger twice and make it spiral out of control. Maybe I'll drop the Nasty Plot/Sludge Wave combo, and add Tailwind like the analysis suggests.

Also if Sludge Wave is removed then the line in the random-teams.js that checks the Sludge Wave/Bomb redundancy will be uneccessary and should be removed